src/pages-and-resources/app-settings-modal/AppSettingsModal.jsx

```diff
@@ -277,23 +279,26 @@ AppSettingsModal.propTypes = {
   title: PropTypes.string.isRequired,
   appId: PropTypes.string.isRequired,
   children: PropTypes.func,
+  bodyChildren: PropTypes.node,
   onSettingsSave: PropTypes.func,
   initialValues: PropTypes.shape({}),
   validationSchema: PropTypes.shape({}),
   onClose: PropTypes.func.isRequired,
   enableAppLabel: PropTypes.string.isRequired,
   enableAppHelp: PropTypes.string.isRequired,
-  learnMoreText: PropTypes.string.isRequired,
+  learnMoreText: PropTypes.string,
```

Comment:
The `learnMoreText` prop is used as the text for the "learn more configuration" link. This link is rendered only if the corresponding documentationLink is provided, and this link is optional. Therefore, the corresponding `learnMoreText` prop should also be optional.